CHANGELOG.md

```diff
@@ -81,6 +81,7 @@
 * [5685](https://github.com/grafana/loki/pull/5685) **chaudum**: Assert that push values tuples consist of string values
 ##### Changes
 * [6042](https://github.com/grafana/loki/pull/6042) **slim-bean**: Add a new configuration to allow fudging of ingested timestamps to guarantee sort order of duplicate timestamps at query time.
+* [6120](https://github.com/grafana/loki/pull/6120) **KMiller-Grafana**: Rename the fudging of ingested timestamps configuration parameter introduced in PR 6042 to avoid using the slang term "fudge."
```

Comment:
I think just saying the change will be more pleasing to end-users. In case they'd like to understand the reasoning behind the change they can always look for the PR description.
```suggestion
* [6120](https://github.com/grafana/loki/pull/6120) **KMiller-Grafana**: Rename fudge_duplicate_timestamp parameter as increment_duplicate_timestamp"
```